### DISCUSSION TITLE
Clear DM memory

### DISCUSSION THREAD

**Discussion by Tragoul** [drizzle-mizzle/CharacterAI-Discord-Bot#56] [Q&A]
Im probably just dumb and cant figure this out.  But how do you clear the characters memory from DMs so a conversation can start over?

**Comment by drizzle-mizzle** [comment]
`reset`

**Comment by drizzle-mizzle** [comment]
![image](https://user-images.githubusercontent.com/55811932/230885751-b00e269c-c7e1-40b1-9974-c095ceb3a679.png)
